### ISSUE TITLE
No tone mapping for DV profile 5 files

### ISSUE THREAD

**Issue by jianglai** [jarnedemeulemeester/findroid#862]
**Describe the bug**
I have a file in DV profile 5, it doesn't have a HDR 10 base layer and my phone (Pixel 9 Pro) does not support DV. With the official Jellyfin app, the server will transcode and tone map the video, but with Findroid, it direct displays the video on the device and the video has a purple/yellow tint.

**To Reproduce**
1. Use a device that does not support DV.
2. Try to play a video with DV profile 5.
3. The video has a purple/yellow tint.

**Expected behavior**
The server transcodes and tone maps, and the client displays video with correct color (no purple/yellow tinting, not washed out).

**Device info (please complete the following information):**
 - Device: Google Pixel 9 Pro
 - Android version: 15
 - App version: 0.15.3
 - Jellyfin version: 10.9.1

**Additional context**

**Comment by florensie** [comment]
I have the same issue

**Comment by realFPS** [comment]
![SmartSelect_20241025_233423_Findroid.png](https://github.com/user-attachments/assets/ffc4a8a1-5af6-43b1-a315-64b10fea0eb2)

I don't think findroid supports transcoding yet.

![SmartSelect_20241025_233423_Findroid.png](https://github.com/user-attachments/assets/4ae42f89-dab6-48c0-a4b4-18ac0fc86563)

Maybe try these settings with mpv?

**Comment by jianglai** [comment]
It's not about having findroid transcode on the client side, but sending the correct client capabilities to the server and have the server transcode to the color space that the client supports.

**Comment by realFPS** [comment]
> It's not about having findroid transcode on the client side, but sending the correct client capabilities to the server and have the server transcode to the color space that the client supports. 

That's what I meant when I said Findroid doesn't support transcoding. I mean to say Findroid does not support server transcoded media yet.

Did you try the mpv settings? You can use mpv to tonemap it on the device, definitely not a permanent solution to this issue but if someone wants to watch DV P5 now then yeah use that.

**Comment by jianglai** [comment]
I tried to set hardware decoding to no, but it just makes the video so choppy that it is unwatchable. I don't think the phone could handle software decoding...

**Comment by realFPS** [comment]
Yeah it's not perfect. Works smoothly only till 1080p stuff and makes the device warm.

**Comment by andretiagogr** [comment]
> I don't think findroid supports transcoding yet.
> 
> ![SmartSelect_20241025_233423_Findroid.png](https://github.com/user-attachments/assets/4ae42f89-dab6-48c0-a4b4-18ac0fc86563)
> 
> Maybe try these settings with mpv?

Automatically disabling hardware decoding for DV Profile 5 on devices that don't support it would be nice. Or just a manual toggle in the player to force software decoding for a specific file
